src/SUnit-Core/TestResult.class.st

```diff
@@ -17,6 +17,41 @@ Class {
 	#category : #'SUnit-Core-Kernel'
 }
 
+{ #category : #'as yet unclassified' }
+TestResult class >> defaultColorBackGroundForErrorTest [
+	^ Smalltalk ui theme testColorError
```

Comment:
Maybe extract `Smalltalk ui` in a separated method to avoid duplicating it everywhere?